api-gateway/src/main/resources/application.yml

```diff
@@ -17,3 +17,8 @@ eureka:
     fetch-registry: true
     service-url:
       defaultZone: http://localhost:8761/eureka
+
+logging:
+  level:
+    org.springframework.cloud.gateway: trace
```

Comment:
This was included for debugging purposes, let's remove the logging configurations.